portfolio/src/main/webapp/index.html

```diff
@@ -100,11 +100,14 @@ <h1>Gallery</h1>
         <div id="fact-container"></div>
       </div>
       </br>
-      <form action="/comment" method="POST">
-        <p>Please leave a comment:</p>
-        <textarea name="title">Enter comment</textarea>
+      <form id="comment-form" class="hidden" method="POST" enctype="multipart/form-data">
+        <p>Please enter a comment:</p>
+        <textarea name="message"></textarea>
```

Comment:
Is the name of this textarea required? I.e. does "message" reference something?

Reply:
Yes. It is used in my doPost function in DataServlet to get the content of the comment.
`final String content = request.getParameter("message");`

Reply:
Ah, I see you changed that in PR #10. In that case, I think at the moment if you submitted PR #10 and _didn't_ submit this PR, the functionality wouldn't work, right?

If so, in the future I think it would be better to change that line in the same PR as this, or to stagger those changes in such a way that the app is still functional after every PR (e.g. first PR = "Update request parameter name to better reflect its usage", second PR = PR #10, third PR = PR #12)

Reply:
Ah i see. Thanks for the advice!